src/structures/DexEntry.ts

```diff
@@ -20,6 +21,9 @@ export default class DexEntry {
   @Field(() => StatsEntry, { description: 'Base stats for a Pokémon' })
   public baseStats!: StatsEntry;
 
+  @Field(() => EvYieldsEntry, { nullable: true, description: 'EV yields for a Pokémon' })
+  public evYields?: EvYieldsEntry;
```

Comment:
```suggestion
  @Field(() => EvYieldsEntry, { description: 'EV yields for a Pokémon' })
  public evYields!: EvYieldsEntry;
```

Once `evYields` has been added everywhere and the `?` in the `pokemon.d.ts` has been removed this change needs to be applied as well